datastream-server/src/main/java/com/linkedin/datastream/server/CoordinatorConfig.java

```diff
@@ -51,6 +52,7 @@ public CoordinatorConfig(Properties config) {
     _retryIntervalMs = _properties.getInt(CONFIG_RETRY_INTERVAL, 1000 /* 1 second */);
     _heartbeatPeriodMs = _properties.getLong(CONFIG_HEARTBEAT_PERIOD_MS, Duration.ofMinutes(1).toMillis());
     _defaultTransportProviderName = _properties.getString(CONFIG_DEFAULT_TRANSPORT_PROVIDER, "");
+    _assignmentChangeThreadPoolThreadCount = _properties.getInt(CONFIG_DEFAULT_ASSIGNMENT_THREAD_COUNT, 1);
```

Comment:
If making the count more than 1 is going to cause an issue, do you think we should just skip reading the value from config? Might want to ignore the config value and deprecate the config?

Reply:
I'm fine with that. I think the original design is to have one thread to per connector, but it requires more change to the thread pool, might be better to fix it into 1.